api/oneclick.py

```diff
@@ -530,16 +560,32 @@ def get_patron_checkouts(self, patron_id):
                 # is included in the downloadUrl, actually
                 acs_resource_id = file.get('acsResourceId', None)
 
-            # TODO: For audio books, the downloads are done by parts, and there are 
-            # multiple download urls.  Need to have a mechanism for putting lists of 
-            # parts into fulfillment objects.
+            # The document at the other end of download_url is a manifest.
+            # For audio books, the manifest includes a list of parts.
+            # For ebooks, the manifest includes a single link to an ACSM
+            # file.
+            #
+            # We can't do anything about audiobook manifests right
+            # now, but we can turn an ebook manifest into a
+            # FulfillmentInfo.
+
+            # We need to use requests instead of self.request because
+            # it's a different server that will reject the credentials
+            # we use for the API. This needs to be mocked separately.
```

Comment:
The previous code assumed that download_url pointed to the ACSM file. But at this point there's a whole other document--the manifest--between us and the ACSM file.